src/libraries/System.Runtime.InteropServices.JavaScript/src/Resources/Strings.resx

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I've seen invisible changes on the first line of a few files in this diff, what are they? Byte order markers?